classwork.html

```diff
@@ -0,0 +1,43 @@
+<html>
```

Comment:
Calculator styling is a bit off (pretty much it is not displayed). Could you please fix it? Also, could you please do all other assignments on separate branch with separate pull requests from now on.